docs/integrations/ai-engines/anomaly.mdx

```diff
@@ -1,23 +1,85 @@
+- See the anomaly detection proposal [document](https://docs.google.com/document/d/1Yd7ARZVg_67xlcY-JR2kuO7mak9Ia2YER1Jk0EdpEa0/edit#heading=h.mo4wxsae6t1d) for more information.
+
+</Info>
+
+<Info>
+**Context about types of anomaly detection**
+
+- Supervised: we have inlier/outlier labels, so we can train a classifier the normal way. This is very similar to a standard classification problem.
+
+- Semi-supervised: we have inlier/outlier labels and perform an unsupervised preprocessing step, and then a supervised classification algorithm.
+
+- Unsupervised: we don’t have inlier/outlier labels and cannot assume all training data are inliers. These methods construct inlier criteria that will classify some training data as outliers too based on distributional traits. New observations are classified against these criteria. However, it’s not possible to evaluate how well the model detects outliers without labels.
+
+</Info>
+
+<Info>
+**Default dispatch logic**
+
+We propose the following logic to determine type of learning:
+- Use supervised learning if labels are available and the dataset contains at least 3000 samples.
+- Use semi-supervised learning if labels are available and number of samples in the dataset is less than 3000.
+- If the dataset is unlabelled, use unsupervised learning.
+
+We’ve chosen 3000 based on the results of the NeurIPS AD Benchmark paper (linked above). The authors report that semi-supervised learning outperforms supervised learning when the number of samples used is less than 5% of the size of the training dataset. The average size of the training datasets in their study is 60,000, therefore this 5% corresponds to 3000 samples on average.
+
+</Info>
+
+<Info>
+**Reasoning for default models on each type**
+
+We refer to the NeurIPS AD Benchmark paper (linked above) to make these choices:
+- For supervised learning, use CatBoost. It often outperforms classic algorithms.
+- For semi-supervised, XGBod is a good default from PyOD.
+- There’s no clear winner for unsupervised methods, it depends on the use case. ECOD is a sensible default with a fast runtime. If we’re not concerned about runtime, we can use an ensemble.
+
+</Info>
+
+## Prerequisites
+
+Before proceeding, ensure the following prerequisites are met:
+
+1. Install MindsDB locally via [Docker](https://docs.mindsdb.com/setup/self-hosted/docker) or [Docker Desktop](https://docs.mindsdb.com/setup/self-hosted/docker-desktop).
+2. To use Anomaly Detection handler within MindsDB, install the required dependencies following [this instruction](https://docs.mindsdb.com/setup/self-hosted/docker#install-dependencies).
 
-# Example usage
+## Setup
+
+Create an AI engine from the [Anomaly Detection handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/anthropic_handler).
+
+```sql
+CREATE ML_ENGINE anomaly_detection_engine
+FROM anomaly_detection;
+```
+
+Create a model using `anthropic_engine` as an engine.
```

Comment:
Should we change the `anthropic_engine`?

Reply:
yes. thank you for catching it.